# A timing stats update

int32     stride  # skipped frames +1


# Timers for different components of the visual measurement pipeline
float32 t_feature_manager
float32 t_homography_manager
float32 t_measurement_generation